diesel_tests/tests/update.rs

```diff
@@ -1,5 +1,9 @@
 use crate::schema::*;
 use diesel::*;
+use diesel::{
+    expression::{bound::Bound, grouped::Grouped, operators},
+    sql_types::{Nullable, Text},
+};
```

Comment:
I would prefer to keep that in line with all other import in diesel. That means: No nested import, but better use the following:
```suggestion
use diesel::expression::{bound::Bound, grouped::Grouped, operators};
use diesel::sql_types::{Nullable, Text};
```